src/ass.js

```diff
@@ -56,13 +56,12 @@ var ASS = (function (m, fs, util, path, mustache)
 
 	ass.render = function(callback)
 	{
-		fs.readFile(path.resolve(__dirname, 'templates/ass.mustache'), 'UTF-8', function(err, result){
-			if(err) {
-				return callback(err);
```

Comment:
Don't want to be difficult.... But... When using this as a library the process might get stuck because the callback is never called.
As you can see it used to do the callback with the error. ^

Reply:
No offense taken, practice makes perfect. Would this be better ?

```javascript
try {
	const result = fs.readFileSync(path.resolve(__dirname, 'templates/ass.mustache'), 'UTF-8');
	callback(null, mustache.render(result, _structured));
} catch (error) {
	console.error("An error occured while processing the template file 'templates/ass.mustache'", error);
	callback(error);
}
```

Reply:
Since you mentionned it, there actually is an issue related to callbacks getting called or not when using this as a lib. I'll open another PR regarding this issue.